### ISSUE TITLE
Editor button on main menu should always be available

### ISSUE THREAD

**Issue by bvschaik** [bvschaik/julius#203]
Right now the editor button on the main menu is only shown if all editor-related files are present (see `editor_is_present()` in `src/editor/editor.c`). This might be confusing. Suggestion: always show the button and if the editor is not present, show a popup with instructions where to get the files.

**Comment by crudelios** [comment]
This could potentially also require specific translations. Either that or the error message is left in plain English.

**Comment by bvschaik** [comment]
True, I will leave it in English, just like the warning when you do not have the patch.
